src/encoded/static/components/PageTitle.js

```diff
@@ -15,6 +15,8 @@ import { content_views } from './globals';
 import { typedefs } from './util';
 import QuickInfoBar from './viz/QuickInfoBar';
 import jsonScriptEscape from './../libs/jsonScriptEscape';
+import { EditableField, FieldSet } from '@hms-dbmi-bgm/shared-portal-components/es/components/forms/components/EditableField';
+import { HiGlassViewConfigTabView } from './../components/item-pages/HiGlassViewConfigView';
```

Comment:
Codacy found an issue: ['HiGlassViewConfigTabView' is defined but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5793828)